src/Compilers/CSharp/Test/Syntax/Syntax/SyntaxNormalizerTests.cs

```diff
@@ -728,6 +728,13 @@ public void TestSpacingOnRecord()
             TestNormalizeDeclaration("record  struct  S(int I, int J);", "record struct S(int I, int J);");
         }
 
+        [Fact]
+        public void TestSpacingOnPrimaryConstructor()
+        {
+            TestNormalizeDeclaration("class  C     (   int    I   ,    int    J   )   ;    ", "class C(int I, int J);");
+            TestNormalizeDeclaration("struct  S     (   int    I   ,    int    J   )   ;    ", "struct S(int I, int J);");
+        }
```

Comment:
>} [](http://example.com/codeflow?start=8&length=1)

Consider also testing `"struct S  (  )  ;"`.

Reply:
> Consider also testing ```"struct S  (  )  ;"```.

Added in
[https://github.com/dotnet/roslyn/pull/66054/files#diff-e1d58451c1f04638587633be9788992449f5eccb1f5fb58cc0ebc54a5e2a6c93](https://github.com/dotnet/roslyn/pull/66054/files#diff-e1d58451c1f04638587633be9788992449f5eccb1f5fb58cc0ebc54a5e2a6c93)